symbols-mui/scripts/generateSymbols.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+GRAY='\033[1;30m'
+NC='\033[0m' # No Color
+
+echo "Building new MUI-style symbols..."
+
+echo -ne "${BLUE}Installing dependencies...${NC}"
+yarn install:dependencies
+echo -e "${BLUE}Done${NC}"
+
+echo -ne "${BLUE}Clearing existing symbols folder...${NC}"
+rm -rf symbols
+mkdir -p symbols
+echo -e "${BLUE}Done${NC}"
+
+echo -ne "Loading latest PX Blue icon package...${NC}"
+yarn upgrade @pxblue/symbols
+rm -rf node_modules/@pxblue/symbols/symbols.svg
+echo -e "${BLUE}Done${NC}"
```

Comment:
These should be pulled from the /symbols folder, not from NPM (you'll also need to build them before you copy them over to make sure they are there and up to date)